tests/common/plugins/custom_skipif/CustomSkipIf.py

```diff
@@ -0,0 +1,18 @@
+from abc import ABCMeta, abstractmethod
+
+
+class CustomSkipIf:
+    __metaclass__ = ABCMeta
+    # def __init__(self, ignore_list, extra_params):
```

Comment:
Suggest uncommenting this function out and having each implementation call `super()`

Reply:
ok, will do fix